tests/JsonQueryWrapper/JsonQueryTest.php

```diff
@@ -73,4 +73,43 @@ public function testSomething()
 
         $this->assertTrue($jq->run('.Foo.Bar == 33'));
     }
+
+    public function testFixProcessBuilderPileup()
+    {
+        $data = json_encode(['Foo' => ['Bar' => 33]]);
+
+        $process1 = $this->getMockBuilder('Symfony\Component\Process\Process')
+            ->disableOriginalConstructor()
+            ->getMock();
+        $process1->expects($this->once())->method('run');
+        $process1->expects($this->once())->method('getOutput')->will($this->returnValue(33));
+
+        $process2 = $this->getMockBuilder('Symfony\Component\Process\Process')
+            ->disableOriginalConstructor()
+            ->getMock();
+        $process2->expects($this->once())->method('run');
+        $process2->expects($this->once())->method('getOutput')->will($this->returnValue(33));
+
+        $pb = $this->getMockBuilder('Symfony\Component\Process\ProcessBuilder')
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Avoid variables with short names like $pb. Configured minimum length is 3.](https://www.codacy.com/app/estahn/json-query-wrapper/file/1949878769/issues/source?bid=3105591&fileBranchId=3107888#l93)